src/kili/entrypoints/mutations/label/__init__.py

```diff
@@ -44,6 +44,7 @@ def create_predictions(
         model_name: Optional[str] = None,
         asset_id_array: Optional[List[str]] = None,
         disable_tqdm: bool = False,
+        overwrite: bool = True,
```

Comment:
I find it risky to change the current default behavior

On the "Explore" UI, it's nice to be able to see the different predictions made by a model over time for example

I would set it to false

Reply:
I agree with you indeed